packages/ui/src/helpers/authenticator/utils.ts

```diff
@@ -180,3 +180,12 @@ export function trimValues<T extends Record<string, string>>(
     {} as T
   );
 }
+
+export const isValidEmail = (value: string | undefined) => {
+  if (!value) return false;
+
+  // source: HTML5 spec https://html.spec.whatwg.org/multipage/input.html#valid-e-mail-address
+  const emailRegex =
```

Comment:
Should this be a constant?

Reply:
Yes, and it is a constant.

Reply:
I mean a `CONSTANT` 😅 , since there is no need to declare this for each call

Reply:
oh gotcha now, makes sense!